css/main.css

```diff
@@ -325,3 +325,130 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+#contact_form {
+  width: 375px;
+  margin: auto;
+  background: #fff;
+  height: 100%;
+}
+
+#contact_form p {
+  margin: 120px 24px 73px 24px;
+  text-align: center;
+  font-weight: 700;
+  font-size: 32px;
+  font-family: "Inter", sans-serif;
+  color: #172b4d;
+  line-height: 44px;
+}
+
+#contact_form p span {
+  display: none;
+}
+
+form {
+  margin-left: 24px;
+  margin-right: 24px;
+  display: flex;
+  flex-direction: column;
+}
+
+form input {
+  padding: 10px;
+  border: none;
+  outline: none;
```

Comment:
- [ ] Good job on styling the contact form but you should make sure your layout and styling matches the original design in [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)
Your Design 
![image](https://user-images.githubusercontent.com/43995830/152349470-4cdbb342-dbb4-4403-9563-aa988d058039.png)

Original (figma) Design
![image](https://user-images.githubusercontent.com/43995830/152349601-112c5bd9-69bd-448b-89f8-c5c4c57fa339.png)